tests/devices/test_default_qubit_jax.py

```diff
@@ -485,7 +485,9 @@ def circuit(weights):
             qml.templates.StronglyEntanglingLayers(weights, wires=[0, 1])
             return qml.expval(qml.PauliZ(0))
 
-        weights = jnp.array(qml.init.strong_ent_layers_normal(n_wires=2, n_layers=2))
+        weights = jnp.array(
+            np.random.random(qml.templates.StronglyEntanglingLayers.shape(n_layers=2, n_wires=2))
+        )

```

Comment:
Here and in the following I am creating numpy arrays and parsing them...the reason to not use native `jax.random` functions is that some tests compare to data produced with a seed, and they fail in such a case.